language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/codeaction/providers/createvar/ErrorHandleOutsideCodeAction.java

```diff
@@ -86,46 +87,53 @@ public List<CodeAction> getCodeActions(Diagnostic diagnostic,
         if (!hasErrorMemberType || nonErrorNonNilMemberCount == 0) {
             return Collections.emptyList();
         }
+        ImportsAcceptor importsAcceptor = new ImportsAcceptor(context);
         List<TextEdit> edits = new ArrayList<>();
-        edits.addAll(getModifiedCreateVarTextEdits(diagnostic, unionTypeDesc, positionDetails,
-                typeSymbol.get(), context));
+        CreateVariableOut modifiedTextEdits = getModifiedCreateVarTextEdits(diagnostic, unionTypeDesc, positionDetails,
+                typeSymbol.get(), context, importsAcceptor);
+        edits.addAll(modifiedTextEdits.edits);
         edits.addAll(CodeActionUtil.getAddCheckTextEdits(
                 PositionUtil.toRange(diagnostic.location().lineRange()).getStart(),
                 positionDetails.matchedNode(), context));
 
         String commandTitle = CommandConstants.CREATE_VAR_ADD_CHECK_TITLE;
-        return Collections.singletonList(CodeActionUtil.createCodeAction(commandTitle, edits, uri,
-                CodeActionKind.QuickFix));
+        int renamePosition = modifiedTextEdits.renamePositions.get(0) - 6;
```

Comment:
Shall we extract this value `6` to a constant?

Reply:
+1